docs/blitz/design/design.rst

```diff
@@ -0,0 +1,1756 @@
+Design
+======
+
+Designed for quick development, the *Blitz* verifies *key-value* pairs and
+uses parsing for full flexibility across OS and platforms. A *blitz datafile* is constructed of testcases.
+Each testcase consists of various `sections` and each section itself contains various `actions`.
+`Actions` are blocks of command that each perform a task as part of the testcase
+and report the results, as `Passed`, `Failed`, `Errored` etc.
+Each section and action will run in the order given within the testcase. 
+
+Blitz yaml
+-----------
+To use the *Blitz* template, add the `YAML` content to a
+*blitz_datafile.yaml* as shown in the following example for `BGP` on a router.
```

Comment:
"oh no - not another yaml file" is what i get when reading blitz datafile. It is the same datafile as a trigger datafile.

To create a Blitz testcase, go to your Trigger datafile, or create a new yaml file if you dont have one, and add below example into it. This example is a BGP ShutNoShut Testcase.

Reply:
Done